x/x.go

```diff
@@ -1135,12 +1135,11 @@ func IsGuardian(groups []string) bool {
 	return false
 }
 
-// RunVlogGC runs value log gc on store. It runs GC unconditionally after every 10 minutes.
-// Additionally it also runs GC if vLogSize has grown more than 1 GB in last minute.
+// RunVlogGC runs value log gc on store. It runs GC unconditionally after every 1 minute.
 func RunVlogGC(store *badger.DB, closer *z.Closer) {
 	defer closer.Done()
 
-	// Runs every 1m, checks size of vlog and runs GC conditionally.
+	// Runs every 1m
 	ticker := time.NewTicker(1 * time.Minute)
 	defer ticker.Stop()
 
```

Comment:
Do you see why this if is present in the code below?
```
if abs(lastSz, sz) > 512<<20 {
```

Reply:
That's a good question. Actually I'm not sure as well. It will certainly reduce the number of logs it outputs, but for what purpose and why 512MiB in particular I don't know. 

This was the commit that introduced that block of code https://github.com/dgraph-io/dgraph/pull/6889/commits/356f246422f261d3d3b7bda54a74ebd3adc83025.

Reply:
seems like it just could be for debugging.